tasks/68_drop-elements/drop-elements.js

```diff
@@ -0,0 +1,14 @@
+// It works but I expected different solution :)
+// I just figure out solution to pass tests...
```

Comment:
probably here you can use `findIndex` function and after slice from this index or if it is -1 return [], it will look shorter

according to methods which proposed in readme you can also use `shift` method until function will return false, but it can be heavy from performance perspective, as when you shift js engine will have to update indexes for all elements